### DISCUSSION TITLE
Browser On RPi Not Starting

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1819] [Fixed Issues]
@rern 
I have just installed a fresh image on my Pi4 and restored a backup and updated it. After the restore, the local screen worked, but after the update to 20241130 it will not start.

**Comment by rern** [comment]
Kernel `linux-rpi` > `6.6.62` breaks it.

Downgrade:
```sh
wget http://tardis.tiny-vps.com/aarm/repos/2024/11/25/aarch64/core/linux-rpi-6.6.62-1-aarch64.pkg.tar.xz
pacman -U linux-rpi-6.6.62-1-aarch64.pkg.tar.xz
```

**Reply by R101G** [reply]
@rern 
Thanks, that seems to have fixed it.
